src/ae/index.js

```diff
@@ -193,6 +195,54 @@ const Ae = stampit(NodePool, Tx, AccountResolver, {
           return handler(...args)
         }
       ]
+    ),
+    ...mapObject(
+      contractMethods,
+      ([name, handler]) => [
+        name,
+        function (...args) {
+          const instanceOptions = {
+            ...this.Ae.defaults,
+            onNode: this.selectedNode.instance,
+            onAccount: this
```

Comment:
let's add `onCompiler` here to be able to call compiler methods from `contractMethods`